gateway-sp-comms/src/single_sp.rs

```diff
@@ -282,15 +284,18 @@ details in `dump.json`."
 pub struct SingleSp {
     interface: String,
     cmds_tx: mpsc::Sender<InnerCommand>,
+    ereport_req_tx: mpsc::Sender<ereport::WorkerRequest>,
```

Comment:
I would not do this as a part of this PR, but I'm curious for your thoughts: there are a bunch of places like this one where we're going from one "thing" (in this case, an `mpsc::Sender`) to two "things". The original "one" is always named generically, and the new one is named indicating it's related to ereports. Do you think we should go back and rename the generic things to indicate they're intended for control-plane-agent?

Reply:
Yeah, I think it might be worthwhile to do that (especially if we ever add a third port to the management network...). You're right that I just didn't really want to touch all the control-plane-agent code in _this_ PR, but I'd be happy to do it in a separate commit.